docs/gen/ex_DIDNOTCONVERGE_msg.rst

```diff
@@ -1,5 +1,5 @@
 .. comment: generated by t_lint_didnotconverge_bad
 .. code-block::
 
-   -V{t#,#}         'stl' region trigger index 1 is active: @([hybrid] a)
+   -V{t#,#}         'stl' region trigger index 1 is active: [hybrid] a
```

Comment:
If you're going to change perhaps quote the 'variable' part of the message?  Although, I think to users the "@" nicely suggests sensitivity to an event, although I realize that's not really correct, so am fine with dropping the @.

```suggestion
   -V{t#,#}         'stl' region trigger index 1 is active: '[hybrid] a'
```